File: program/program/check-leap-year/CheckLeapYear.cs
using System;  
namespace CheckLeapYear  
{  
    class Program   
    {  
        static void Main(string[] args)  
        {  
            Console.WriteLine("Enter a Year : ");  
            int Year = int.Parse(Console.ReadLine());  
            if (((Year % 4 == 0) && (Year % 100 != 0)) || (Year % 400 == 0)) Console.WriteLine("{0} is a Leap Year.", Year);  
            else Console.WriteLine("{0} is not a Leap Year.", Year);  
            Console.ReadLine();  
        }  
    }  
}  

File: program/program/check-leap-year/CheckLeapYear.java
import java.util.Scanner;

public class CheckLeapYear {

  public static void main(String[] args) {
    System.out.print("Enter year : ");
    Scanner sc = new Scanner(System.in);
    int year = sc.nextInt();
    sc.close();

    if ((0 == year % 4) && (0 != year % 100) || (0 == year % 400)) {
      System.out.println(year + " is a leap year.");
    } else {
      System.out.println(year + " is not a leap year.");
    }
  }
}
